apps/src/aichat/views/chatMessage.module.scss

```diff
@@ -4,6 +4,7 @@
   padding: 10px 10px;
   border-radius: 10px;
   overflow: hidden;
+  white-space: pre-line;
```

Comment:
Just curious how you decided to select `pre-line` vs the other possible options.(e.g. `pre-wrap`).

Reply:
Honestly I wasn't sure which was better :) One concrete thing is that I noticed the bot responses seem like they often (always?) have a space at the start, which `pre-line` strips (and `pre-wrap` does not).

![image](https://github.com/code-dot-org/code-dot-org/assets/25372625/2b516f1e-8d42-443c-a40e-1d24df6df2cb)